packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -111,6 +128,62 @@ describe('internal-plugin-metrics', () => {
       assert.deepEqual(cdl.getMeetingInfoReqResp(), 10);
     });
 
+    describe('measureLatency', () => {
+      let clock;
+      let callbackStub;
+      let saveLatencySpy;
+
+      beforeEach(() => {       
+        clock = sinon.useFakeTimers(); 
+        
+        saveLatencySpy = sinon.stub(cdl, 'saveLatency');
+      });
+
+      afterEach(() => {
+        clock.restore();
+        sinon.restore();
+      });
+      
+      it('checks measureLatency with overwrite false', async () => {
+        const key = 'internal.client.pageJMT';
+        const overwrite = false;
+        callbackStub = sinon.stub().resolves();
+
+        const promise = cdl.measureLatency(callbackStub, 'internal.client.pageJMT', overwrite);
+        clock.tick(50);
+
+        await promise;
```

Comment:
issue: This needs to check the promise resolution is the expected resolution from the callback.